jicofo/rootfs/defaults/logging.properties

```diff
@@ -1,7 +1,9 @@
-{{ if .Env.SENTRY_DSN | default "0" | toBool }}
+{{ $SENTRY_DSN := .Env.SENTRY_DSN | default .Env.JICOFO_SENTRY_DSN -}}
```

Comment:
This does not help. JICOFO_SENTRY_DSN is not evaluated (as only SENTRY_DSN is)

Reply:
I would suggest to just strip the ` | default "0" | toBool`-part.

This is at least working in my environment